Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/WPF/InkToolbar/InkToolbar.cs

```diff
@@ -0,0 +1,192 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Windows;
+using System.Windows.Markup;
+using Microsoft.Windows.Interop;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.WPF
+{
+    [ContentProperty(nameof(Children))]
+    public class InkToolbar : WindowsXamlHost
+    {
+        public global::Windows.UI.Xaml.Controls.InkToolbar UwpControl => this.XamlRoot as global::Windows.UI.Xaml.Controls.InkToolbar;
```

Comment:
>UwpControl [](start = 59, length = 10)

Public fields are bad (I'll stop repeating this on each type, but it's present on many)